src/Cmfd.cpp

```diff
@@ -447,10 +424,10 @@ void Cmfd::computeDs(int moc_iteration) {
                     * d * f + next_length_perpen * d_next*f_next);
 
             /* Compute net current */
-            current = sense * _surface_currents[cell*_num_cmfd_groups*8 +
-                      surface*_num_cmfd_groups + e] - sense
-                      * _surface_currents[cell_next*_num_cmfd_groups*8 +
-                      next_surface*_num_cmfd_groups + e];
+            current = sense * _surface_currents->getValue
```

Comment:
Personally, I find lines like these easier to comprehend when they are broken up into multiple lines, even at the expense of a few new ("temporary") variables.